app/builtin-pages/com/downloads-list.js

```diff
@@ -61,13 +61,18 @@ export function render (downloadsList) {
             <i class="fa fa-times"></i>
           </button>
         `
-
+        if (process.platform == "win32"){
```

Comment:
I am guessing you've checked this, but just to be sure the "real" node process is in scope here right?  i.e. it has the process.platform property?

Also a small nit: probably safer to use === instead of ==

Reply:
true